purchases/src/main/kotlin/com/revenuecat/purchases/listenerConversions.kt

```diff
@@ -132,53 +128,51 @@ fun Purchases.purchaseProductWith(
 }
 
 /**
- * Purchase a [Product]'s [PurchaseOption].
+ * Make a purchase upgrading from a previous sku.
```

Comment:
one more place to call out that if it's a subscription, the default option will be chosen?